app/helpers/permissions_helper.rb

```diff
@@ -0,0 +1,9 @@
+module PermissionsHelper
```

Comment:
Do we need this any more?

Reply:
@duncanjbrown thanks I've removed the old helper implementation and squashed down to https://github.com/DFE-Digital/apply-for-teacher-training/pull/2388/commits/73f442863a0ab3648c1584bd1a5e003e31c0257b